testcharms/charm-repo/quantal/lxd-profile-subordinate/lxd-profile.yaml

```diff
@@ -0,0 +1,39 @@
+#name: juju-default
+description: lxd profile subordinate for testing
+config:
+#
+# allowed config
+#
+  security.nesting: "false"
+  security.privileged: "true"
+  linux.kernel_modules: openvswitch,nbd,ip_tables,ip6_tables,iptable_nat
+  environment.http_proxy: ""
+#
+# blacklisted config
+#
+# boot.autostart: "true"
+# limits.... 
+# migration... 
+devices:
+#
+# allowed devices
+#
+  sandisk:
```

Comment:
It'd be helpful to have comments on the keys which will provide something new to the machines lxd profile and which we expect to supersede keys in the lxd-profile charm.